cypher/cypher-docs/src/docs/dev/syntax/expressions.asciidoc

```diff
@@ -93,16 +106,16 @@ include::../ql/query-syntax-label.adoc[leveloffset=+1]
 
 
 [[syntax-list-label]]
-=== List of `LABEL` expressions examples
+=== List of label expressions examples
 
-The following table displays whether the `LABEL` expression matches the node:
+The following table displays whether the label expression matches the node:
 
 [cols="^3,^2,^2,^2,^2,^2,^2,^2,^2"]
 |===
 |
 8+^|*Node*
 
-|*`LABEL` expression* | `()` | `(:A)` | `(:B)` | `(:C)` | `(:A:B)` | `(:A:C)` | `(:B:C)` | `(:A:B:C)`
+|*Label expression* | `()` | `(:A)` | `(:B)` | `(:C)` | `(:A:B)` | `(:A:C)` | `(:B:C)` | `(:A:B:C)`
```

Comment:
Is there a reason why the corresponding table is not done for relationship type expressions? This would be very useful. (If this _has_ been done, then please ignore this comment -- I must obviously have missed it...)

Reply:
The idea was that it could be used for both relationship and label expressions, but now that i think about it we cannot use some expressions for relationship so i can try and create an additional one for relationship type expressions